tests/core/engine_adapter/integration/test_integration_clickhouse.py

```diff
@@ -0,0 +1,522 @@
+import typing as t
+import pytest
+from tests.core.engine_adapter.integration import TestContext
+import pandas as pd
+from sqlglot import exp, parse_one
+
+
+@pytest.fixture(
+    params=[
+        pytest.param(
+            "clickhouse",
+            marks=[
+                pytest.mark.docker,
+                pytest.mark.engine,
+                pytest.mark.clickhouse,
+                pytest.mark.xdist_group("engine_integration_clickhouse"),
```

Comment:
As of [e8576e9a](https://github.com/TobikoData/sqlmesh/commit/e8576e9aab71d3237a80b360d8ae1ed417849540) the `xdist_group`'s arent required anymore

Reply:
Nice - i see some additional ones in the dbt tests, should I put up a PR that just removes those?

Reply:
Oh, good point. I was focusing on the integration tests in `test_inetgration.py` when I removed the `xdist_group`'s. 

Probably can leave the DBT ones for now, there isn't a speed issue with those tests